stubs/requests/requests/utils.pyi

```diff
@@ -47,6 +52,7 @@ def set_environ(env_name: str, value: None) -> AbstractContextManager[None]: ...
 def should_bypass_proxies(url: _Uri, no_proxy: Iterable[str] | None) -> bool: ...
 def get_environ_proxies(url: _Uri, no_proxy: Iterable[str] | None = ...) -> dict[Any, Any]: ...
 def select_proxy(url: _Uri, proxies: Mapping[Any, Any] | None): ...
+def resolve_proxies(request: Request | PreparedRequest, proxies: Mapping[str, str], trust_env: bool = ...): ...
```

Comment:
Looks like `proxies` can also be `None`: https://github.com/psf/requests/blob/177dd90f18a8f4dc79a7d2049f0a3f4fcc5932a0/requests/utils.py#L868

```suggestion
def resolve_proxies(request: Request | PreparedRequest, proxies: Mapping[str, str] | None, trust_env: bool = ...): ...
```